code/renderers/vue3/src/index.ts

```diff
@@ -6,4 +6,10 @@ export * from './public-api';
 export * from './public-types';
 
 // optimization: stop HMR propagation in webpack
-if (typeof module !== 'undefined') module?.hot?.decline();
+try {
+  if (module?.hot?.decline) {
+    module.hot.decline();
+  }
+} catch (e) {
+  console.log('  e:', e);
```

Comment:
Do we need to log this?

Reply:
not need to log but we need to wrap it in try catch. , it will break in vite ( it is webpack specific )

Reply:
Okay, let's remove the console.log then